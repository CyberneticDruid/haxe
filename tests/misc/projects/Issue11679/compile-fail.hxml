-main Main
-D message.no-color
