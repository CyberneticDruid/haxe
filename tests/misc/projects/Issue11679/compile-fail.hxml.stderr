[ERROR] Bar.hx:1: characters 1-2

 1 | @:build(Macro.build())
   | ^
   | Unexpected &

